docs/guide/extensibility/indentation.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Indentation
+---
+
+# Indentation
+
+Sublime Text can perform automatic indentation when you type.
+It can also reindent text according to the indentation rules on demand.
+Note that it will not perform auto-formatting, like adding newlines or adjusting spaces around parenthesis and the like. You would need an external tool or plugin for that.
+The indentation rules consist of various regular expression patterns applied in different circumstances.
+All the regular expression patterns match against one line of text.
+
+## File Format
+
+Indentation rules are defined using metadata files.
+However, because metadata for indentation rules is complicated and
+is commonly required by packages which add syntax definitions,
+it's discussed separately in this page
```

Comment:
As a general style thing, I tend to refrain from using shortenings like "it's" in prose. But this is an optional thing and just an FYI. I will most likely do a full pass on the page once it's ready for a more detailed review and commit it directly.